pytorch_lightning/tuner/auto_gpu_select.py

```diff
@@ -30,14 +32,14 @@ def pick_multiple_gpus(nb):
 
     nb = torch.cuda.device_count() if nb == -1 else nb
 
-    picked = []
+    picked = []  # type: List[int]
```

Comment:
does it work to do this?
```suggestion
    picked: List[int] = []
```

Reply:
yea I think this is [better](https://mypy.readthedocs.io/en/stable/getting_started.html#local-type-inference) i'll use this here. (just saw some comment syntax `# type: ignore` in the codebase)